File: src/main/java/liqp/filters/H.java
package liqp.filters;

import liqp.nodes.RenderContext;

class H extends Filter {

    /*
     * h(input)
     *
     * Alias for: escape
     */
    @Override
    public Object apply(RenderContext context, Object value, Object... params) {
      final Escape escape = (Escape) getFilter("escape");
      return escape.apply(context, value, params);
    }
}

File: liqp-core/src/main/kotlin/liqp/filter/H.kt
package liqp.filter

import liqp.context.LContext
import liqp.params.FilterParams

class H : LFilter() {

  /**
   * h(input)
   *
   * Alias for: escape
   */
  override fun onFilterAction(context: LContext, value: Any?, params: FilterParams): Any? {
    val escape:Escape by context.parseSettings.filters
    return escape.onFilterAction(context, value, params)
  }
}

